### ISSUE TITLE
Windows xp, windows vista, and windows7 speech recognition

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1273]
**Reported by jkenn337 on 2010-12-11 16:03**
Script NVDA so it has full support for windows xp, vista, and windows7 speech recognition. this includes the speech recognition training dialogs.

**Comment by leonardder** [comment]
@DerekRiemer and @JCSTeh: since we have dictation bridge now, I guess we can close this? Or is there anything that still can and should be done at the NVDA side of things?

**Comment by derekriemer** [comment]
Yeah. Dictationbridge is very robust here. For future readers of this ticket, you can get it at http://dictationbridge.com
Closing as fixed!
